block_producer_swap_bot/cli/generic/help.py

```diff
@@ -0,0 +1,9 @@
+"""
+Provide help messages for command line interface's commands.
+"""
+
+ETH_PROVIDER_ARGUMENT_HELP_MESSAGE = 'Ethereum provider to pull swap requests from'
+REMNODE_ARGUMENT_HELP_MESSAGE = 'Remnode host and port. For example 127.0.0.1:8888'
+PERMISSION_ARGUMENT_HELP_MESSAGE = 'A block producer account and permission' \
+                                   ' of level to authorize, as in \'account@permission\''
+PRIVATE_KEY_ARGUMENT_HELP_MESSAGE = 'Block producer\'s private key to sign blocks.'
```

Comment:
lets use a separate producer permission to approve swaps

Reply:
Done.